BrowserKit/Sources/ComponentLibrary/ContextualHintView/ContextualHintView.swift

```diff
@@ -27,10 +27,12 @@ public class ContextualHintView: UIView, ThemeApplicable {
     // MARK: - UI Elements
     private lazy var contentContainer: UIView = .build { _ in }
 
-    private lazy var closeButton: ActionButton = .build { button in
+    private lazy var closeButton: UIButton = .build { button in
```

Comment:
I can hear @lmarceau already, telling you to use the `LinkButton` instead 🤣 It's actually a small change from what you already have now.

Reply:
Yes, I know, but that one is a Resizable button too and it did not work. Maybe if a take a closer look.

Reply:
I have changed that. It should be updated in main.